tests/template_tests/filter_tests/test_wordwrap.py

```diff
@@ -89,3 +89,14 @@ def test_wrap_long_text(self):
             "I'm afraid",
             wordwrap(long_text, 10),
         )
+
+    def test_wrap_preserve_newlines(self):
+        long_text = (
```

Comment:
Perhaps we should use a variable name that does not imply that the length is important. How about just `text` or `value`?